<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="encrypt_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_txt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Affine Cypher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decrypt_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="a_value">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="b_value">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="a_txt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>a :</string>
   </property>
  </widget>
  <widget class="QLabel" name="b_txt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>b :</string>
   </property>
  </widget>
  <widget class="QLabel" name="instruction_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Enter your message here :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_msg_value">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>126</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="output_msg_txt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Your final message :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="output_msg_value">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="instruction_msg1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Warning: you must enter values such as a is an odd integer different</string>
   </property>
  </widget>
  <widget class="QLabel" name="a_check">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="b_check">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="instruction_msg1_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>than 13 and b must be between 0 and 25 included</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
